python/cugraph-dgl/tests/nn/test_gatconv.py

```diff
@@ -23,18 +26,17 @@
 torch = import_optional("torch")
 dgl = import_optional("dgl")
 
-options = {
-    "idtype_int": [False, True],
-    "max_in_degree": [None, 8],
-    "num_heads": [1, 3],
-    "to_block": [False, True],
-}
+options = OrderedDict(
+    {
+        "idtype_int": [False, True],
+        "max_in_degree": [None, 8],
+        "num_heads": [1, 3],
+        "to_block": [False, True],
+    }
+)
 
 
-@pytest.mark.parametrize("to_block", options["to_block"])
-@pytest.mark.parametrize("num_heads", options["num_heads"])
-@pytest.mark.parametrize("max_in_degree", options["max_in_degree"])
-@pytest.mark.parametrize("idtype_int", options["idtype_int"])
+@pytest.mark.parametrize(",".join(options.keys()), product(*options.values()))
```

Comment:
That's a neat way to do this.

Do you need `OrderedDict` though above, since dictionary order is preserved in default `dict` types since python 3.7?